PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/runner/webapp/config/RiderWebAppConfiguration.kt

```diff
@@ -52,13 +56,7 @@ class RiderWebAppConfiguration(project: Project, factory: ConfigurationFactory,
         return RiderWebAppRunState(project, myModel)
     }
 
-    override fun onNewConfigurationCreated() {
-        super.onNewConfigurationCreated()
-
-        if (name.equals(UNNAMED, true)) {
-            name = this.factory?.type?.displayName ?: UNNAMED
-        }
-    }
+    override fun suggestedName(): String? = SUGGESTED_NAME_PREFIX + " - " + project.name
```

Comment:
This will add a solution name to run configuration. The idea was to have a run configuration name based on the project selected in a project combo box. When a user select a different value we automatically update the configuration name, so, user could have a multiple run configurations for each project (since publish logic persists per project). If user change the value in the run configuration name manually, we ignore all the logic above and use user's value. I'm not sure if we can access run configuration name from an editor. So, that could be a bummer here :) but need to try that first.

Reply:
Fixed, can you check once more?